ouroboros-consensus-cardano/changelog.d/20240328_165135_alexander.esgen_milestone_11_gdd_governor_squashed.md

```diff
@@ -0,0 +1,7 @@
+### Non-Breaking
+
+- Set Genesis window for Byron and Shelley.
```

Comment:
We should either use past tense here, or present in the other fragments :)

Reply:
I think "Set ..." is conveniently both present and past tense :sweat_smile: